vassal-app/src/main/java/VASSAL/build/GpIdChecker.java

```diff
@@ -51,6 +51,7 @@ public class GpIdChecker {
 
   protected GpIdSupport gpIdSupport;
   protected int maxId;
+  public int noGpIdMatch = 0; // shared to GameRefresher
```

Comment:
The only classes which should have public fields are ones which are _solely_ data classes.